README.md

```diff
@@ -5,7 +5,39 @@ Create Ruby clients from a protocol buffer description of an API.
 **Note** This project is a preview. Please try it out and let us know what you think,
 but there are currently no guarantees of stability or support.
 
-## Usage
+## Getting started using the Docker image
+The easiest way to use the generator is to run it with the Docker image.
```

Comment:
It's probably important to note that the docker image is actually specific to the `gapic-generator-cloud` generator.

Reply:
done